### DISCUSSION TITLE
Codeblock in after Mermaid Diagram

### DISCUSSION THREAD

**Discussion by interas99** [requarks/wiki#3533] [Error / Bug Report]
### Actual behavior

When creating a 'mermaid' diagram, a code block is always displayed additionally on the finished page. (Markdown Editor)

### Expected behavior

Code block should not appear

### Steps to reproduce the behavior

- Create new Page with Markdown-Editor
- Add a simple Mermaid Diagramm. e.g.:

```
classDiagram
class Square~Shape~{
    int id
    List~int~ position
    setPoints(List~int~ points)
    getPoints() List~int~
}`
```

- create page and you will see the code block (it´s not visible in the Editor)

Preview in Editor:
![bug_editor_view](https://user-images.githubusercontent.com/34246253/84652512-01a0b880-af0c-11ea-817a-e0e267c16e8d.png)

Page after exit Editor:
![bug_finish_view](https://user-images.githubusercontent.com/34246253/84652515-02394f00-af0c-11ea-8335-5319fa6a2774.png)

The Bug appears after upgrading from 2.3 to 2.4

Current used Version: 2.4.107 on Ubuntu

**Comment by interas99** [comment]
Update: I´ve noticed that this "Bug" only appears when I´m adding that Diagram inside a Tab in the new tabset:

Replace '!!!' with code chars (3x '`')

```
## Header 1 {.tabset}
### Header 2

some text

#### <span style="color:#FF6A00"> Header inside the container </span>
some text

!!!mermaid
sequenceDiagram
    Alice ->> Bob: Hello Bob, how are you?
    Bob-->>John: How about you John?
    Bob--x Alice: I am good thanks!
    Bob-x John: I am good thanks!
    Note right of John: Bob thinks a long<br/>long time, so long<br/>that the text does<br/>not fit on a row.

    Bob-->Alice: Checking with John...
    Alice->John: Yes... John, how are you?
!!!
```

**Reply by naseer** [reply]
I have the same problem but it seems to always happen.

**Reply by naseer** [reply]
I am on version 2.5.197
